src/main/java/main/Aragorn.java

```diff
@@ -0,0 +1,177 @@
+package main;
+
+import exceptions.AragornException;
+import commands.inputParser;
+import tasks.*;
+import java.util.Scanner;
+
+public class Aragorn {
+
+    private static final String LINE =  "    __________________________________________________________\n";
+    private static final String GREET = "    Hello! I am main.Aragorn son of Arathorn, and am called Elessar, the Elfstone, Dúnadan,\n" +
+            "    the heir of Isildur Elendil's son of Gondor.\n" +
+            "    What can I do for you?\n";
+    private static final String EXIT = "    Bye. Hope to see you again soon!\n";
+    private static final String TAB = "    ";
+
+    private static final String commandList = "    Here is a list of commands:\n" +
+            "\n" +
+            "    \"list\": Displays list of tasks.\n" +
+            "\n" +
+            "    \"todo <description>\": Adds a Todo task to the list.\n" +
+            "\n" +
+            "    \"deadline <description> /by <deadline>\": Adds a task and its deadline to the list.\n" +
+            "\n" +
+            "    \"event <description> /from <start> /to <end>\": Adds an event and its start and end conditions to the list.\n" +
+            "\n" +
+            "    \"mark <task number>\": Marks the corresponding task in the list as completed.\n" +
+            "\n" +
+            "    \"unmark <task number>\": Marks the corresponding task in the list as incomplete.\n" +
+            "\n" +
+            "    \"/help\": Displays this list of commands.\n" +
+            "\n" +
+            "    \"bye\": Closes the program.\n";
+
+    public static void main(String[] args) throws AragornException {
+        Task[] list = new Task[100];
+        int listLength = 0;
+        int remainingTasks = 0;
+        System.out.println(LINE + GREET + LINE);
+        int index;
+        String icon;
+        Scanner in = new Scanner(System.in);
+        while(true) {
+            String userInput = in.nextLine();
+            String commandType = inputParser.commandIdentifier(userInput);
+            try {
+                inputParser input = new inputParser(userInput.trim(), commandType);
+
+
+                switch (commandType) {
+                    case "LIST":
+                        if (listLength == 0) {
```

Comment:
this is 4 levels of nesting, it makes the code hard to read and debug if any problems arise. consider refactoring this to multiple methods.